### ISSUE TITLE
[Bug] full finetune wan2.1-1.3b, generate noisy videos

### ISSUE THREAD

**Issue by liuuzexiang** [hao-ai-lab/FastVideo#1102]
### Describe the bug

Hi, 
    I try to finetune the wan2.1-1.3B model, 
but after training 200 steps, the validation generated video is all noisy. Is there some bugs?
Load the official wan2.1-1.3B to inference before training, the validation video is normal.

### Reproduction

bash examples/training/finetune/wan_t2v_1.3B/crush_smol/finetune_t2v.sh

### Environment

torch 2.10.0
cuda_12.6.r12.6/compiler.35059454_0

**Comment by liuuzexiang** [comment]
the loss is small.

<img width="3690" height="1512" alt="Image" src="https://github.com/user-attachments/assets/1d2eaba8-b279-4364-a5d7-9810409e90fc" />
